src/sagemaker/jumpstart/validators.py

```diff
@@ -109,6 +178,10 @@ def validate_hyperparameters(
         region (str): Region for which to validate hyperparameters. (Default: JumpStart
           default region).
 
+    Raises:
+        JumpStartHyperparametersError: If the hyperparameter is not formatted correctly,
```

Comment:
non-blocking: should `hyperparameter` be singular? The function takes a dict of hyperparameters.